backend/infrahub/trigger/tasks.py

```diff
@@ -7,16 +7,22 @@
 from prefect.client.orchestration import PrefectClient
 from prefect.client.schemas.filters import DeploymentFilter, DeploymentFilterName
 
-from .catalogue import triggers
-from .constants import DEPRECATED_STATIC_TRIGGER_NAMES
+from infrahub.trigger.models import TriggerDefinition
+
+# from .catalogue import triggers
 from .models import TriggerType
 
 if TYPE_CHECKING:
     from uuid import UUID
 
 
-@task(name="trigger-setup", task_run_name="Setup triggers in task-manager", cache_policy=NONE)  # type: ignore[arg-type]
-async def setup_triggers(client: PrefectClient) -> None:
+@task(name="trigger-setup", task_run_name="Setup triggers of type {trigger_type.value}", cache_policy=NONE)  # type: ignore[arg-type]
+async def setup_triggers(
```

Comment:
Converted this function to work with all types of trigger Builtin, Workflow etc ...